src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    @Override
+    public List<Truck> get() {
+        List<Truck> trucks = new ArrayList<>();
+        trucks.add(new Truck()); // Додаємо новий Truck до списку
```

Comment:
The `get` method currently adds only one `Truck` to the list. If the task requires producing multiple trucks, consider adding more instances to the list.